packages/ui-tests/cypress/support/next-commands/design.ts

```diff
@@ -100,3 +100,30 @@ Cypress.Commands.add('selectCamelRouteType', (type: string, subType?: string) =>
   subType = subType ?? type;
   cy.get(`[data-testid="new-entity-${subType}"] button.pf-v5-c-menu__item`).click({ force: true });
 });
+
+Cypress.Commands.add('selectRuntimeVersion', (type: string) => {
+  cy.hoverOnRuntime(type);
+  cy.get(`[data-testid^="runtime-selector-Camel ${type}"] button.pf-v5-c-menu__item`).click({ force: true });
+  cy.waitSchemasLoading();
+
+  cy.get('[data-testid="visualization-empty-state"]').should('exist');
+  // Wait for the element to become visible
+  cy.get('[data-testid="visualization-empty-state"]').should('be.visible');
```

Comment:
I do not see an explicit wait. So etiher there is no wait or it is by default of the cy.get and in both cases I think we can remove the comment

Reply:
thanks for pointing this out - removed the comment